packages/protocol/contracts-0.8/common/CeloDistributionSchedule.sol

```diff
@@ -41,6 +41,14 @@ contract CeloDistributionSchedule is UsingRegistry, ReentrancyGuard, Initializab
     _;
   }
 
+  modifier onlyEpochManager() {
```

Comment:
I'm thinking we should completely throw away this contract.

Nothing of the current logic remains. Instead what we have is a CeloUnreleasedTreasury contract that holds unreleased CELO and only allows the Epoch manager to release it with some `releaseXXX()` methods

Reply:
done